src/profile/webid.test.ts

```diff
@@ -302,6 +318,53 @@ describe("getProfileAll", () => {
     );
   });
 
+  it("returns an array of the objects of triples of the WebID doc such as <webid, rdfs:seeAlso, ?object>", async () => {
+    const mockedFetch = jest
+      .fn<(typeof CrossFetch)["fetch"]>()
+      .mockResolvedValueOnce(
+        mockResponse(
+          await triplesToTurtle(toRdfJsQuads(MOCK_PROFILE)),
+          {
+            headers: {
+              "Content-Type": "text/turtle",
+            },
+          },
+          "https://some.profile"
+        )
+      )
+      .mockResolvedValueOnce(
+        mockResponse(
+          await triplesToTurtle(toRdfJsQuads(MOCK_PROFILE)),
+          {
+            headers: {
+              "Content-Type": "text/turtle",
+            },
+          },
+          "https://some.other.profile"
+        )
+      );
+    const profileContent = buildThing({ url: MOCK_WEBID })
+      .addIri(rdfs.seeAlso, "https://some.profile")
+      .addIri(rdfs.seeAlso, "https://some.other.profile")
+      .build();
+
+    const webIdProfile = setThing(
+      mockSolidDatasetFrom(MOCK_WEBID),
+      profileContent
+    );
+    const result = await getProfileAll(MOCK_WEBID, {
+      fetch: mockedFetch,
+      webIdProfile,
+    });
+    expect(result.altProfileAll).toHaveLength(2);
+    expect(getThingAll(result.altProfileAll[0])).toStrictEqual(
+      getThingAll(MOCK_PROFILE)
+    );
+    expect(getThingAll(result.altProfileAll[1])).toStrictEqual(
+      getThingAll(MOCK_PROFILE)
+    );
+  });
```

Comment:
Perhaps add another test where both `seeAlso` and `isPrimaryTopicOf` are both used on the same WebId; and check the results array is returned in the expected order?

Reply:
Do you mean checking that duplicates are removed?

Reply:
Yep :)

Reply:
Ignore previous reply - I thought I was responding to the above discussion.

My understanding of the ticket is that `rdfs:seeAlso` should be prioritized over `foaf:isPrimaryTopicOf` and so this should be reflected in the order of the objects in the array returned. We should have a couple of checks for this in the tests to avoid regressions.